explorer/parse_and_execute/parse_and_execute_test.cpp

```diff
@@ -36,9 +37,9 @@ TEST(ParseAndExecuteTest, Recursion) {
       ParseAndExecute("explorer/data/prelude.carbon", "test.carbon", source,
                       /*parser_debug=*/false, &trace_stream, &llvm::nulls());
   ASSERT_FALSE(err.ok());
-  EXPECT_THAT(err.error().message(),
-              Eq("RUNTIME ERROR: overflow:1: stack overflow: too many "
```

Comment:
Shouldn't this still be tagged as a runtime error? Part of validating that was to ensure it's consistent with the typical error flow.

Reply:
It is still tagged as a runtime error, but it also includes a real filename and line number now, which will be different depending on exactly what we set the stack limit to. I can check for a prefix of `RUNTIME ERROR:` as well if you like?

Reply:
I'd prefer that, yes. You could switch to a regex match FWIW

Reply:
Done.